requirements.txt

```diff
@@ -1,12 +1,12 @@
-Django==3.2.5
-djangorestframework==3.12.4
-django-auth-ldap==2.4.0
+Django==3.2.11
+djangorestframework==3.13.1
+django-auth-ldap==4.0.0
```

Comment:
Tested auth-ldap locally? That is the one big thing which is not currently automatically tested.

Reply:
I haven not (apart from running its test suite); but I read the changelog and don't expect any surprises:

*  https://github.com/django-auth-ldap/django-auth-ldap/blob/master/docs/changes.rst
*  https://github.com/django-auth-ldap/django-auth-ldap/compare/2.4.0...4.0.0

Adding integration tests for LDAP connectivity should not be Too Hard, methinks...  I'll open an issue for it.